lib/Sema/CSApply.cpp

```diff
@@ -4389,7 +4404,7 @@ namespace {
       
       // The final component type ought to line up with the leaf type of the
       // key path.
-      assert(!baseTy || baseTy->getWithoutSpecifierType()->isEqual(leafTy));
+      assert(!baseTy|| baseTy->getWithoutSpecifierType()->isEqual(leafTy));
```

Comment:
Did you mean to change the whitespace here?

Reply:
No it was inadvertent. I'll fix it.